node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -151,16 +160,32 @@ where
     vertices_to_be_added: PendingVertices<I, C>,
     /// The duration for which incoming vertices with missing dependencies are kept in a queue.
     pending_vertex_timeout: TimeDiff,
+    /// Instance ID of an era for which this synchronizer is constructed for.
+    era_id: C::InstanceId,
+    /// Keeps track of the lowest/oldest seen unit per validator when syncing.
+    /// Used only for logging.
+    oldest_seen_panorama: ValidatorMap<Option<u64>>,
+    /// Boolean flag indicating whether we're synchronizing current era.
+    pub(crate) current_era: bool,
 }
 
 impl<I: NodeIdT, C: Context + 'static> Synchronizer<I, C> {
     /// Creates a new synchronizer with the specified timeout for pending vertices.
-    pub(crate) fn new(pending_vertex_timeout: TimeDiff) -> Self {
+    pub(crate) fn new(
+        pending_vertex_timeout: TimeDiff,
+        validator_len: usize,
+        era_id: C::InstanceId,
+    ) -> Self {
         Synchronizer {
             vertex_deps: BTreeMap::new(),
             vertices_to_be_added_later: BTreeMap::new(),
             vertices_to_be_added: Default::default(),
             pending_vertex_timeout,
+            oldest_seen_panorama: ValidatorMap::from(
+                iter::repeat(None).take(validator_len).collect_vec(),
+            ),
```

Comment:
I think we have a `FromIterator` impl for this:
```suggestion
            oldest_seen_panorama: iter::repeat(None).take(validator_len).collect(),
```